exchange/gdpr.go

```diff
@@ -3,6 +3,8 @@ package exchange
 import (
 	"encoding/json"
 
+	gpplib "github.com/prebid/go-gpp"
+	gppConstants "github.com/prebid/go-gpp/constants"
```

Comment:
FYI: In the next GPP lib iteration, I wonder if the constants package is really necessary. Perhaps that should be brough up into the gpp level.

Reply:
That has been on my mind. If any of the sections need access to the constants, it needs to be separate. I figured it was easier to merge it later than split it later once the need, or lack thereof, was clearer.

Reply:
Ok, cool.